packages/api/src/controllers/asset.ts

```diff
@@ -182,7 +184,7 @@ function parseUrlToDStorageUrl(
 }
 
 export async function validateAssetPayload(
-  req: Pick<Request, "body" | "user" | "token" | "config">,
+  req: Pick<Request, "body" | "user" | "token" | "config" | "project">,
```

Comment:
Shouldn't this be `projectId`?

Reply:
I was thinking we pull the project from the db and add it to the request like we do for the user object

Reply:
`project` here makes sense I think?

Reply:
Ohh I thought this was getting an asset, but it's the request object. `project` is the right thing yeah